tools/html_transformer/html_transformer.js

```diff
@@ -0,0 +1,95 @@
+var Promise = require('es6-promise').Promise;
+var jsdom = require('jsdom');
+
+exports.window = null;
+
+exports.load = function(filename) {
+  return new Promise(function(resolve, reject) {
+    jsdom.env(filename, function(errors, window) {
+      if (errors && errors.length) {
+        return reject(errors[0]);
```

Comment:
I hope this doesn't mean we can only see one error at a time. This would make fixing the errors a little bit frustrating by having to run N times for N errors present before resolving all of them.